setup.py

```diff
@@ -32,6 +32,7 @@
     "zarr",
     "requests",
     "defusedxml",
+    "umap-learn",
```

Comment:
do we need to add this to requirements files as well?

Reply:
I am not sure where this is being used - I merged from develop

Reply:
ok I can see it is already in requirements files so ignore this comment.